lib/shrinkwrap-package.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var Package = require('./package');
+
+function ShrinkwrapPackage(name, versionSpecified, versionInstalled, parents) {
```

Comment:
The `updateRequired` logic is going to be different for each type of dependency. That bower and npm package deps are largely the same is more a coincidence than a rule.

We did not want to leak the `updateRequired` logic into the checker at:

* https://github.com/quaertym/ember-cli-dependency-checker/pull/26/files#diff-2dffa192d080ff64d8bdd0f869f82621R14

The checker itself just gathers information. Similar to the main dependency checker, it doesn't presume to make a decision about what actions must occur.